frontend/components/branchViewLink.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
This is the new component for use in both `SiteGithubBranchesTable` and `SitePublishedBranchesTable`